RakutenShop/Scenes/RecipeDetailView/RecipeDetailViewCoordinator.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  RecipeDetailViewCoordinator.swift
+//  RakutenShop
+//
+//  Created by Garnik Ghazaryan on 2/11/20.
+//  Copyright © 2020 Garnik Ghazaryan. All rights reserved.
+//
+
+import UIKit
+import ReactiveSwift
+
+class RecipeDetailViewCoordinator: BaseCoordinator {
+
+    private let context: Context
+    private var viewController: RecipeDetailViewController?
+    var recipeDetailCoordinator: StepListViewCoordinator?
+    var recipeObject: RecipeObject?
+    
+    init(context: Context, coordinator: BaseCoordinator) {
+        self.context = context
+        super.init(coordinator: coordinator)
+        self.viewController = R.storyboard.main.recipeDetailViewController()
+        guard let viewController = viewController else { return }
+        let viewModel = RecipeDetailViewViewModel(context: context, coordinatorDelegate: self)
+        viewController.viewModel = viewModel
+    }
+    
+    override func start() {
+        guard let viewController = viewController, let rootNavController = rootViewController as? UINavigationController else { return }
+        rootNavController.pushViewController(viewController, animated: true)
```

Comment:
it would be greate to have push function which is organizing this in the base classes, look the example in the Ihor's project.